frontend/src/pages/offers/ViewPallet.tsx

```diff
@@ -90,11 +100,6 @@ const ViewPallet: FunctionComponent<Props> = ({
             </p>
             <ul className="list-disc list-inside">
               <li>a pallet can contain at most 1 bulk bag</li>
-              <li>a pallet can contain a maximum of 36 boxes</li>
-              <li>
-                if a pallet contains a bulk bag, it can also contain a maximum
-                of 18 boxes
-              </li>
```

Comment:
**question:** same question as above. Do we not want to enforce those restrictions anymore?

Reply:
Yes, this limit was based on banana boxes, but other types of boxes must be possible. This was flagged in test feedback: https://www.notion.so/distributeaid/1-pallet-can-contain-any-number-of-boxes-containers-depends-on-the-size-of-the-container-Is-1-36-23d67a3ab6464a09aa3aee100a727676